src/cargo/lib.rs

```diff
@@ -1,6 +1,7 @@
 #![deny(unused)]
-#![feature(collections, hash, io, libc, os, path, std_misc, unicode, env, core)]
-#![cfg_attr(test, deny(warnings))]
+#![feature(collections, hash, libc, os, std_misc, unicode, env, core)]
+#![feature(old_io, old_path, str_words)]
+//#![cfg_attr(test, deny(warnings))]
```

Comment:
There are deprecation warnings with `CString::from_{vec,slice}` that are not easily fixable and would need quite a lot of work currently and I reckon that would be easier with the new io/path/os/fs in plac.